tools/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 
 FROM build AS install
 ARG LEVELDB=1.20
-ARG ROCKSDB=6.24.2
+ARG ROCKSDB=7.9.2
```

Comment:
Latest version is now v8 BTW

Reply:
https://github.com/facebook/rocksdb/releases/tag/v8.0.0

Reply:
https://github.com/linxGnu/grocksdb/pull/111
Need a few tweaks to support v8.0

Reply:
you rock!